### PR TITLE
Add Play1 v1.6.0 release

### PR DESCRIPTION
**Pull request by xael-fry**

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
At least one pull request committer is not linked to a user. See https://help.github.com/en/articles/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user

File: conf/playReleases.json [hunk 1]
```diff
@@ -286,6 +286,12 @@
             "name": "Streamy",
             "version": "2.5.0",
             "date": "Mar 4 2016"
+        },
+		{
```
Comment (mkurz) [inline conf/playReleases.json:290]: @xael-fry Please fix this opening bracket, thanks!

**Comment by lightbend-cla-validator** [comment]
At least one pull request committer is not linked to a user. See https://help.github.com/en/articles/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user

**Comment by lightbend-cla-validator** [comment]
At least one pull request committer is not linked to a user. See https://help.github.com/en/articles/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user

File: conf/playReleases.json (hunk 1 above)
Comment (xael-fry) [reply conf/playReleases.json]: fix it

**Pull request merged**